src/Products/Annotation/Annotator/BaseAnnotator.cs

```diff
@@ -12,93 +13,92 @@ namespace GroupDocs.Annotation.MVC.Products.Annotation.Annotator
     public abstract class BaseAnnotator
     {
         public string Message = "Annotation of type {0} for this file type is not supported";       
-        protected AnnotationDataEntity annotationData;
-        protected PageData pageData;
+        protected static AnnotationDataEntity annotationData;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'annotationData' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/groupdocs-annotation/GroupDocs.Annotation-for-.NET-MVC/pullRequest?prid=5372471)